celery/backends/elasticsearch.py

```diff
@@ -132,7 +137,9 @@ def _get_server(self):
             '%s:%s' % (self.host, self.port),
             retry_on_timeout=self.es_retry_on_timeout,
             max_retries=self.es_max_retries,
-            timeout=self.es_timeout
+            timeout=self.es_timeout,
+            scheme=self.scheme,
```

Comment:
I guess this is `http` by default, correct?

Reply:
Yes, I added a unit test to check that.